specs/agents/metadata.md

```diff
@@ -58,9 +61,12 @@ or equal to `U+0020` (space).
 
 Agents MAY use alternative approaches, but those need to generally conform to the basic concept. Failing to discover the 
 proper hostname may cause failure in correlation between APM traces and data reported by other clients (e.g. 
-Metricbeat). For example, if the agent uses an API that produces the FQDN, this value is likely to mismatch hostname 
+Metricbeat). For example, if the agent uses an API that produces host name (not FQDN), this value is likely to mismatch hostname 
 reported by other clients.
 
+Starting from APM Server 8.9, APM agents should report the fully qualified domain name (FQDN). Before 8.9 APM agents
+should report only the host name (first part of FQDN until the first dot `.`).
```

Comment:
I wonder if we could drop this requirement. Just always prefer reporting the FQDN for now. Part of Silvia's argument was that host.name is little used in APM, so isn't considered a breaking change.

Reply:
I removed this requirement in last commit.